can/logconvert.py

```diff
@@ -0,0 +1,69 @@
+"""
+Convert a log file from one format to another.
+"""
+
+import sys
+import argparse
+import errno
+
+from can import LogReader, Logger, SizedRotatingLogger
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        "python -m can.logconvert",
```

Comment:
I don't think that we need this, do we?

```suggestion
```

Reply:
No, not really. But I added it to keep consistency with the other scripts. I think it should be here until all the scripts are cleaned up.

Reply:
👍

I'll remove that line in a separate PR real quick so you can just apply this suggestion.

Reply:
See #1099